.github/workflows/qa-rpc-integration-tests.yml

```
@@ -16,6 +16,16 @@ on:
       - synchronize
       - ready_for_review
 
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: ${{ !contains(fromJSON('[
+      "refs/heads/release/2.60",
+      "refs/heads/release/2.61",
+      "refs/heads/release/3.0",
+      "refs/heads/main"
+    ]'), github.ref) }}
```

Please check if this works, would prevent us needing to add every single release branch

```suggestion
  cancel-in-progress: >-
    ${{
      !startsWith(github.ref, 'refs/heads/release/') && 
      github.ref != 'refs/heads/main'
    }}
```